src/components/layout/footer.tsx

```diff
@@ -6,6 +6,8 @@ import MainLinksSection from './footer/main-links'
 import DisclaimerSection from './footer/disclaimer'
 import { Container } from 'components/containers'
 import { DerivStore } from 'store'
+import Branding from 'components/containers/branding'
```

Comment:
export branding from index.ts so you can import it from `components/containers`

Reply:
okay